### PR TITLE
[ANDWL-206] dialog-listview-refactoring

### PR DESCRIPTION
**Pull request by kimheeje12**

### CODE REVIEW

File: app/src/main/java/wannabit/io/cosmostaion/dialog/Dialog_Currency_Set.java [hunk 1]
```diff
@@ -26,215 +38,72 @@ public static Dialog_Currency_Set newInstance(Bundle bundle) {
...
+                }
+            });
+        }
+
+        @Override
+        public int getItemCount() {
+            return mSymbolList == null ? 0 : mSymbolList.length;
+        }
+
+        public class ChainListHolder extends RecyclerView.ViewHolder {
```
Comment (Kwonhyukjoon) [inline app/src/main/java/wannabit/io/cosmostaion/dialog/Dialog_Currency_Set.java:91]: Replace holder name

File: app/src/main/java/wannabit/io/cosmostaion/dialog/Dialog_Currency_Set.java [hunk 2]
```diff
@@ -26,215 +38,72 @@ public static Dialog_Currency_Set newInstance(Bundle bundle) {
...
-            public void onClick(View v) {
-                Intent resultIntent = new Intent();
-                resultIntent.putExtra("currency", 16);
-                getTargetFragment().onActivityResult(getTargetRequestCode(), Activity.RESULT_OK, resultIntent);
-                getDialog().dismiss();
+    }
 
-            }
-        });
+    private BaseActivity getSActivity() {
```
Comment (Kwonhyukjoon) [inline app/src/main/java/wannabit/io/cosmostaion/dialog/Dialog_Currency_Set.java:106]: Do you need it?

File: app/src/main/java/wannabit/io/cosmostaion/dialog/Dialog_Currency_Set.java (hunk 1 above)
Comment (kimheeje12) [reply app/src/main/java/wannabit/io/cosmostaion/dialog/Dialog_Currency_Set.java:91]: YES SIR!

File: app/src/main/java/wannabit/io/cosmostaion/dialog/Dialog_Currency_Set.java (hunk 2 above)
Comment (kimheeje12) [reply app/src/main/java/wannabit/io/cosmostaion/dialog/Dialog_Currency_Set.java:106]: I'll delete!

Comment (kimheeje12) [reply app/src/main/java/wannabit/io/cosmostaion/dialog/Dialog_Currency_Set.java:106]: > Do you need it?

I'll delete!

File: app/src/main/java/wannabit/io/cosmostaion/dialog/Dialog_Currency_Set.java (hunk 1 above)
Comment (kimheeje12) [reply app/src/main/java/wannabit/io/cosmostaion/dialog/Dialog_Currency_Set.java:91]: > Replace holder name

YES SIR!

**Review by soaryong-c — APPROVED** [review]
👍

**Pull request merged**